Update transforms.md (#26644)
TheOdinProject/curriculum: advanced_html_css/animation/transforms.md

### Three-dimensional transforms

> The `rotate`, `scale`, and `translate` transform functions aren't limited to just the 2D plane. They work for the 3D space as well! However, to perceive a 3D effect on some of these function values, `perspective` is required.

Reviewer: The rest of the lesson uses the wording 3D plane, so I think it would be confusing to use the word space here. What do you think about changing it to something like this:
```suggestion
The `rotate`, `scale`, and `translate` transform functions aren't limited to just 2D planes. They also work for 3D planes as well! However, to perceive a 3D effect on some of these function values, `perspective` is required.
```

Author: A plane typically refers to a flat 2D object, whereas when discussing 3D transformations, it's more accurate to use the term '3D space.' I've gone ahead and replaced all instances of '3D plane' with '3D space' in the file. I hope this adjustment is appropriate and will improve the clarity of the content.

#### Rotate

> These are the additional transform function values to rotate an element on a 3D space:

Reviewer: ```suggestion
These are the additional transform function values to rotate an element in a 3D space:
```

#### Scale

> These are the additional transform function values to scale an element on a 3D space:

Reviewer: ```suggestion
These are the additional transform function values to scale an element in a 3D space:
```

#### Translate

> These are the additional transform function values to translate an element on a 3D space:

Reviewer: ```suggestion
These are the additional transform function values to translate an element in a 3D space:
```
